src/truflation/data/exporter.py

```diff
@@ -158,6 +158,10 @@ def reconcile_dataframes(df_base: pandas.DataFrame, df_incoming: pandas.DataFram
         # Exclude 'created_at' from merge identifiers
         identifiers = [x for x in df_base.columns if x not in ['created_at']]
 
+        # Convert 'value' to have same types
+        df_base['value'] = df_base['value'].astype(object)
+        df_incoming['value'] = df_incoming['value'].astype(object)
```

Comment:
why do we need that?

Reply:
to make the export to be flexible as in, we have `float/string` to pass as value, but the backend always have `double[pyarrow]` type, it will output error even though the value is the same.

Reply:
let's move that into another PR, we need to check that exporter with the transformers and all ingestors before deploying it

Reply:
the problem with removing this lies within export itself,

the database expects `double[pyarrow]` type
the gsheets expects `object` type

so when, I try to export to db and to gsheet, it cannot merge